doc/Quick-Start-Guide.rst

```diff
@@ -114,10 +114,11 @@ Unit Benchmark
 
 Troubleshooting
 ================
-- If the pip installation fails while installing build dependencies,
-  run the following commands::
+- If pip fails to install a package while installing build dependencies,
+  try to manually install the package.
```

Comment:
can we rephrase to install it using pip3 insted of manually installing.
"try installing packages using pip installer"

Reply:
Done.